core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -1020,7 +1020,7 @@ public void syncPersistDirectory(InodeDirectory dir, JournalContext journalConte
           }
         }
       } else {
-        if (!retry.attemptRetry()) {
+        if (!retry.attempt()) {
```

Comment:
This usage goes against our pattern, do you mind fixing it or leaving a TODO and creating a JIRA ticket to make this consistent?

Reply:
I will fix it, thanks!